mod/merge_users/actions/users/merge.php

```diff
@@ -163,6 +200,15 @@
   unset($old_user->gc_skills);
 }
 
+//Test for deactivate
+$deactivate = get_input('deactivate');
+
+if($deactivate){
+  $old_user->gcdeactivate = true;
+  $old_user->gcdeactivatereason = "";
```

Comment:
This can take a string so we can put "merge" or something here so we can keep track.